src/mds/LogSegment.h

```diff
@@ -33,13 +36,33 @@ class CDentry;
 class MDSRank;
 struct MDSlaveUpdate;
 
-typedef uint64_t log_segment_seq_t;
```

Comment:
It would be preferrable to move this into the LogSegment class. A `using` directive is also better:

`using seq_t = uint64_t;`

Let's update the uses of `log_segment_seq_t` in the small bits of `.cc` code to do `LogSegment::seq_t` instead.

Reply:
I have updated the patch.